ArdusatSDK.h

```diff
@@ -315,10 +315,13 @@ class Orientation: public Sensor {
 
     boolean read(Acceleration & accel, Magnetic & mag);
 
-    const char * toCSV(const char * sensorName);
-    const char * toJSON(const char * sensorName);
+    const char * readToCSV(const char * sensorName);
```

Comment:
Yes, I tested it and it seems to work fine. I noticed that because I added a `readToCSV(Acceleration & accel, Magnetic & mag, const char * sensorName)`, i got errors that `readToCSV(const char * sensorName)` was unimplemented for Orientation. But I have tested both functions and they work.

Note: I also noticed that adding `virtual` to `Sensor::readToCSV` and `Sensor::readToJSON` added  ~350 bytes to the all_sensors sketch.